tests/backup_e2e.rs

```diff
@@ -0,0 +1,27 @@
+// This is an integration test file that lives in the tests/ directory
```

Comment:
comment on filename - avoid abbreviations, they can be harder to follow

call it `end_to_end_tests.rs` for now